src/css/index.css

```diff
@@ -0,0 +1,62 @@
+body {
+  margin: 0;
+  font-family:
+    -apple-system,
+    BlinkMacSystemFont,
+    'Segoe UI',
+    'Roboto',
+    'Oxygen',
+    'Ubuntu',
+    'Cantarell',
+    'Fira Sans',
+    'Droid Sans',
+    'Helvetica Neue',
+    sans-serif;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+}
+
+code {
+  font-family:
+    source-code-pro,
+    Menlo,
+    Monaco,
+    Consolas,
+    'Courier New',
+    monospace;
+}
+
+.remove-btn {
+  color: var(--clr-primary);
+  letter-spacing: var(--spacing);
+  cursor: pointer;
+  font-size: 0.85rem;
+  background: transparent;
+  border: none;
+  margin-top: 0.375rem;
+  transition: var(--transition);
+}
+
+.remove-btn:hover {
+  color: var(--clr-primary-light);
+}
+
+.btn {
```

Comment:
- [ ] Kindly consider making the padding of your button and the font color match the given template.
#### Your design
![image](https://user-images.githubusercontent.com/94551484/222884752-15a15434-cea6-492a-a162-27077e348d4f.png)
### GIven template
![image](https://user-images.githubusercontent.com/94551484/222884764-b57b6fba-e463-4d30-85ae-7ff3cf5428d7.png)